kclvm/compiler_base/span/src/span.rs

```diff
@@ -2,4 +2,9 @@ use rustc_span;
 
 pub type BytePos = rustc_span::BytePos;
 pub type Span = rustc_span::Span;
+pub type SpanData = rustc_span::SpanData;
 pub const DUMMY_SP: Span = rustc_span::DUMMY_SP;
+
+pub fn new_byte_pos(arg: u32) -> rustc_span::BytePos {
```

Comment:
more documents on the fuction and make it #[inline]?

Reply:
fixed